tests/units/oup/test_oup_parser.py

```diff
@@ -36,14 +35,40 @@ def test_dois(parsed_articles):
 def test_no_doi_article(shared_datadir, parser):
     article_name = "ptac108_without_doi.xml"
     with open(shared_datadir / article_name) as file:
-        content = ET.fromstring(file.read())
+        content = parse_without_names_spaces(file.read())
         with raises(RequiredFieldNotFoundExtractionError):
             parser._publisher_specific_parsing(content)
 
 
 def test_no_doi_value_article(shared_datadir, parser):
     article_name = "ptac108_without_doi_value.xml"
     with open(shared_datadir / article_name) as file:
-        content = ET.fromstring(file.read())
+        content = parse_without_names_spaces(file.read())
         with raises(RequiredFieldNotFoundExtractionError):
             parser._publisher_specific_parsing(content)
+
+
+def test_journal_doc_types(parsed_articles):
+    doc_types = ["article", "article", "article"]
```

Comment:
would be nice to have an article in other type there

Reply:
I can try to find one

Reply:
managed to find just 1